add self-signed certificat procedures to secure map and centreon web (#607)
centreon/centreon-documentation: en/administration/secure-platform.md

By default, Centreon installs a web server in HTTP mode. It is strongly recommended to switch to HTTPS mode by adding your certificate.

> It is better to use a certificate validated by an authority rather than a self-signed one. However, in case self-signed method suits you more, you can refer to the [appropriate section](#Securing-the-Apache-web-server-with-auto-signed-certificat)

Reviewer: ```suggestion
It is better to use a certificate validated by an authority rather than a self-signed one. However, in case self-signed method suits you more, you can refer to the [appropriate section](#Securing-the-Apache-web-server-with-self-signed-certificat)
```

```
<!--END_DOCUSAURUS_CODE_TABS-->

> ### Securing the Apache web server with auto-signed certificat

Reviewer: ```suggestion
### Securing the Apache web server with self-signed certificat
```

Let's assume that we have a Centreon server with a `centreon7.localdomain` FQDN address.

1. Preparation of the openssl configuration

Due to a policy change at google, self-signed certificates may be rejected by the google chrome browser. (it is not even possible to add an exception). To continue to use this browser, you have to change the openssl configuration.

open the file `/etc/pki/tls/openssl.cnf` and find the `[v3_ca]` section:
```text
# Add the alt_names tag that allows you to inform our various IPs and FQDNs for the server
[ alt_names ]
IP.1 = xxx.xxx.xxx.xxx
DNS.1 = centreon7.localdomain
# If you have several IP (HA: vip + ip)
# IP.2 = xxx.xxx.xxx.xxx
[ v3_ca ]
subjectAltName = @alt_names
```

2. Creating a private key for the server

Let's create a private key nammed `centreon7.key` without a password so that it can be used by the apache service.
```text

> openssl genrsa -out centreon7.key 1024

Reviewer: ```suggestion
openssl genrsa -out centreon7.key 2048
```

```

Protect your file by limiting rights:
```text
chmod 400 centreon7.key
```

3. Creation of a certificate signing request file 

From the key you created, create a CSR (Certificate Signing Request) file. Fill in the fields according to your company. The "Common Name" field must be identical to the hostname of your apache server (in our case it is centreon7.localdomain).
```text
openssl req -new -key centreon7.key -out centreon7.csr
```

4. Creation of a private key for the certificate authority's certificate

> First, create a private key for this authority. We add the -des3 option to include a password. This password will be requested each time this key is used.

Reviewer 2: ```suggestion
First, create a private key for this authority. We add the -aes256 option to encrypt the output key and include a password. This password will be requested each time this key is used.
```

> First, create a private key for this authority. We add the -des3 option to include a password. This password will be requested each time this key is used.

Reviewer 2: ```suggestion
First, create a private key for this authority. We add the -aes256 option to encrypt the output key and include a password. This password will be requested each time this key is used.
```

```text

> openssl genrsa -des3 1024 > ca_demo.key

Reviewer: ```suggestion
openssl genrsa -des3 2048 > ca_demo.key
```

> openssl genrsa -des3 2048 > ca_demo.key

Reviewer 2: ```suggestion
openssl genrsa -aes256 2048 > ca_demo.key
```

> openssl genrsa -des3 2048 > ca_demo.key

Reviewer 2: ```suggestion
openssl genrsa -aes256 2048 > ca_demo.key
```
